src/components/Card/VerticalCard/VerticalCard.jsx

```diff
@@ -25,7 +25,10 @@ function VerticalCard({ product }) {
   }
 
   return (
-    <div className="card-basic card_custom_width app">
+    <div
+      className="card-basic card_custom_width app"
+      onClick={() => navigate(`/singleProduct/${_id}`)}
```

Comment:
As this navigation happens on click, using `Link` would be more suitable.

Reply:
Yess , navigate also okay , in documentation people use these way too.